src/components/localization/language-switcher.js

```diff
@@ -38,12 +41,12 @@ class LanguageSwitch extends Component {
     handleSelect = (e) => {
         const { id } = e.target
         const current_lang = localStorage.getItem('i18n') || 'en'
-        const path = id === '/en' ? '/' : id
+        const path = id === '/en/' ? '/' : id
```

Comment:
can it ever be `/en` here?

Reply:
we used this plugin `gatsby-plugin-force-trailing-slashes`. so it will always have trailing slashes in the end 🙏